vpr/src/base/read_options.h

```diff
@@ -133,6 +133,7 @@ struct t_options {
     argparse::ArgValue<float> place_crit_limit;
     argparse::ArgValue<int> place_constraint_expand;
     argparse::ArgValue<bool> place_constraint_subtile;
+    argparse::ArgValue<constraints_split_factor> floorplan_split;
```

Comment:
could move write_vpr_constraints and floorplan_split to be general options instead of placer options (since they don't actually affect how the placer works)